packages/loaders/src/elements/Progress.tsx

```diff
@@ -12,9 +12,11 @@ import { StyledProgressBackground, StyledProgressIndicator } from '../styled';
 const COMPONENT_ID = 'loaders.progress';
 
 interface IProgressProps extends HTMLAttributes<HTMLDivElement> {
-  /** The progress as a value between 0 and 100 */
+  /** Sets the progress as a value between 0 and 100 */
   value?: number;
+  /** Sets the foreground bar fill `color`. Defaults to the `successHue` theme value */
   color?: string;
+  /** Sets the `height` of the progress bar. Can be: “small”, “medium”, or “large” */
```

Comment:
The usage of the backticks in this description confused me initially. Does the word “height” in this description refer to the CSS attribute or the characteristic of the `Progress` element? 

I read this as it referring the characteristic of an element, particularly because “small”, “medium”, and “large” aren’t valid values for CSS `height` attribute. 

Reply:
On alternative for this description could be:

> Sets the height of the progress bar.

This re-phrase does two things. 1) height now refers to a characteristic of the element, rather than a CSS code value. 2) "small", "medium", and "large" is not communicated twice since it will be communicated in the `type` section of this prop. Curious what ya'lls thoughts are.

Reply:
I think in this case it actually does set the `height` but I agree, I like prioritizing the characteristic over the property. Will amend. 

The second part you called out is something I wanted to ask the group (from the [API Template doc](https://docs.google.com/document/d/19QvlF4MDKFfUKzHeUfrKIS9856GyTWvAXJwoSKRKX-Q/edit#heading=h.70kaxhzb08a8)). It's a little forward thinking and will result in some initial redundancy but[ this is what I'm proposing the formatting in the long game might be.](https://share.getcloudapp.com/P8uE0wLk) Would be curious if we could make it better.

Reply:
> I think in this case it actually does set the height

Oh yes, it does set the `height`. However `large`, `medium`, etc isn't a valid `height` value. I think the prop description can communicate that the visual height changed, and not necessarily refer to the CSS attribute `height` — which initially confused me since it sentence read like `height` was a characteristic of the element but it also had code backticks. For example, the visual height can be changed using the `transform` property. Then it could be

> Sets the `transform: scaleY` of the progress bar.

but instead of exposing the implementation details in the description, it could be phrased as:

> Sets the height of the progress bar.

And the underlying implementation can use `height`, `transform`, etc.